torchvision/models/detection/transform.py

```diff
@@ -193,7 +195,7 @@ def __repr__(self):
 
 def resize_keypoints(keypoints, original_size, new_size):
     # type: (Tensor, List[int], List[int])
-    ratios = [float(s) / float(s_orig) for s, s_orig in zip(new_size, original_size)]
+    ratios = [torch.scalar_tensor(s) / torch.scalar_tensor(s_orig) for s, s_orig in zip(new_size, original_size)]
```

Comment:
can you clarify the difference between `scalar_tensor` and `tensor` with a scalar value? IIRC `scalar_tensor` is an internal API and non documented

Reply:
I don't see any major difference between the two. I don't see any issues on export side when using tensor. I'm trying to keep these changes consistent within the code and looks like scalar_tensor was mainly used for dynamic export of scalars.

Reply:
if there are no major differences, can we instead use `tensor`, given that `scalar_tensor` is an internal API?

Reply:
same as here, maybe passing `float32` dtype would be better